Bottom: a5ba49c68245d61eaa9038fa596d3092dfdc2a12
Top:    bfdb76c7bd1e0a0ed0a7edbb19bffac4d563b9f4
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-10 15:14:14 -0500

debian backup script

Update backup script to debian:latest

---

diff --git a/roles/docker-common/templates/after_backup.j2 b/roles/docker-common/templates/after_backup.j2
index 4e9fb0f..c501943 100644
--- a/roles/docker-common/templates/after_backup.j2
+++ b/roles/docker-common/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up docker backup dir
-docker run --rm -v {{ docker_backup_dir }}/..:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/docker"
+docker run --rm -v {{ docker_backup_dir }}/..:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/docker"
 {% endif %}
diff --git a/roles/docker-common/templates/before_backup.j2 b/roles/docker-common/templates/before_backup.j2
index 3a843e7..044a7af 100644
--- a/roles/docker-common/templates/before_backup.j2
+++ b/roles/docker-common/templates/before_backup.j2
@@ -2,6 +2,6 @@
 
 # *****************************************************************************
 # Make docker base backup dir
-docker run --rm -v {{ docker_backup_dir }}/..:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/docker"
+docker run --rm -v {{ docker_backup_dir }}/..:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/docker"
 mkdir -p -m u=rwx,g=rwx,o= {{ docker_backup_dir }}
 {% endif %}
diff --git a/roles/docker-exim4/templates/after_backup.j2 b/roles/docker-exim4/templates/after_backup.j2
index bbe68c1..b9c64d9 100644
--- a/roles/docker-exim4/templates/after_backup.j2
+++ b/roles/docker-exim4/templates/after_backup.j2
@@ -1,6 +1,6 @@
 {% if exim4_docker_backup_dir is defined %}
 
 # *****************************************************************************
-# Clean up exim4
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/EXIM4"
+# Clean up exim4 backup destination
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ exim4_image_name }}:{{ docker_image_tag }} bash -c "rm -rf /tmp/import_export/EXIM4"
 {% endif %}
diff --git a/roles/docker-exim4/templates/before_backup.j2 b/roles/docker-exim4/templates/before_backup.j2
index 14e8055..5bb9165 100644
--- a/roles/docker-exim4/templates/before_backup.j2
+++ b/roles/docker-exim4/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # Backup EXIM data container
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/EXIM4"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ exim4_image_name }}:{{ docker_image_tag }} bash -c "rm -rf /tmp/import_export/EXIM4"
 mkdir -p -m u=rwx,g=rwx,o= {{ exim4_docker_backup_dir }}
 docker run --rm \
     --volumes-from {{ exim4_dv_name }} \
diff --git a/roles/docker-gitlab/templates/after_backup.j2 b/roles/docker-gitlab/templates/after_backup.j2
index 11b9797..85b103f 100644
--- a/roles/docker-gitlab/templates/after_backup.j2
+++ b/roles/docker-gitlab/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up git
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/GIT"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/GIT"
 {% endif %}
diff --git a/roles/docker-gitlab/templates/before_backup.j2 b/roles/docker-gitlab/templates/before_backup.j2
index d85e979..d304bcb 100644
--- a/roles/docker-gitlab/templates/before_backup.j2
+++ b/roles/docker-gitlab/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # Backup gitlab
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/GIT"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/GIT"
 mkdir -p -m u=rwx,g=rwx,o= {{ gitlab_docker_backup_dir }}
 pushd {{ docker_restore_config_base_dir }}/{{ gitlab_dv_name }}
 ./gitlab.sh backup
diff --git a/roles/docker-mantisbt/templates/after_backup.j2 b/roles/docker-mantisbt/templates/after_backup.j2
index 5df9695..a853420 100644
--- a/roles/docker-mantisbt/templates/after_backup.j2
+++ b/roles/docker-mantisbt/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up mantisbt
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/MANTISBT"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/MANTISBT"
 {% endif %}
diff --git a/roles/docker-mantisbt/templates/before_backup.j2 b/roles/docker-mantisbt/templates/before_backup.j2
index ced1090..2f9735c 100644
--- a/roles/docker-mantisbt/templates/before_backup.j2
+++ b/roles/docker-mantisbt/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # Backup MantisBT data container
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/MANTISBT"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/MANTISBT"
 mkdir -p -m u=rwx,g=rwx,o= {{ mantisbt_docker_backup_dir }}
 pushd {{ docker_restore_config_base_dir }}/{{ mantisbt_dv_name }}
 ./mantisbt.sh backup
diff --git a/roles/docker-mediawiki/templates/after_backup.j2 b/roles/docker-mediawiki/templates/after_backup.j2
index 23a5485..5988a3c 100644
--- a/roles/docker-mediawiki/templates/after_backup.j2
+++ b/roles/docker-mediawiki/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up wiki
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/WIKI"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/WIKI"
 {% endif %}
diff --git a/roles/docker-mediawiki/templates/before_backup.j2 b/roles/docker-mediawiki/templates/before_backup.j2
index 71ebb80..4983fc1 100644
--- a/roles/docker-mediawiki/templates/before_backup.j2
+++ b/roles/docker-mediawiki/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # Backup WIKI data container
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/WIKI"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/WIKI"
 mkdir -p -m u=rwx,g=rwx,o= {{ wiki_docker_backup_dir }}
 pushd {{ docker_restore_config_base_dir }}/{{ wiki_dv_name }}
 ./mediawiki.sh backup
diff --git a/roles/docker-openldap/templates/after_backup.j2 b/roles/docker-openldap/templates/after_backup.j2
index 86cf502..fde0880 100644
--- a/roles/docker-openldap/templates/after_backup.j2
+++ b/roles/docker-openldap/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up ldap
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/LDAP"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/LDAP"
 {% endif %}
diff --git a/roles/docker-openldap/templates/before_backup.j2 b/roles/docker-openldap/templates/before_backup.j2
index 1b15bfa..00145ae 100644
--- a/roles/docker-openldap/templates/before_backup.j2
+++ b/roles/docker-openldap/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # backup OpenLDAP data container
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/LDAP"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/LDAP"
 mkdir -p -m u=rwx,g=rwx,o= {{ openldap_docker_backup_dir }}
 docker run --rm \
     --volumes-from {{ openldap_dv_name }} \
diff --git a/roles/docker-openssl/templates/after_backup.j2 b/roles/docker-openssl/templates/after_backup.j2
index 4a6fa06..b299e0e 100644
--- a/roles/docker-openssl/templates/after_backup.j2
+++ b/roles/docker-openssl/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up openssl
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/openssl"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/openssl"
 {% endif %}
diff --git a/roles/docker-openssl/templates/before_backup.j2 b/roles/docker-openssl/templates/before_backup.j2
index 160ff32..3120162 100644
--- a/roles/docker-openssl/templates/before_backup.j2
+++ b/roles/docker-openssl/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # Backup openssl data container
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/openssl"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/openssl"
 mkdir -p -m u=rwx,g=rwx,o= {{ openssl_docker_backup_dir }}
 docker run --rm \
     --volumes-from {{ openssl_dv_name }} \
diff --git a/roles/docker-svn/templates/after_backup.j2 b/roles/docker-svn/templates/after_backup.j2
index f134ce9..690c0b2 100644
--- a/roles/docker-svn/templates/after_backup.j2
+++ b/roles/docker-svn/templates/after_backup.j2
@@ -2,5 +2,5 @@
 
 # *****************************************************************************
 # Clean up svn
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/SVN"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/SVN"
 {% endif %}
diff --git a/roles/docker-svn/templates/before_backup.j2 b/roles/docker-svn/templates/before_backup.j2
index 94ab6b1..2f88c25 100644
--- a/roles/docker-svn/templates/before_backup.j2
+++ b/roles/docker-svn/templates/before_backup.j2
@@ -2,7 +2,7 @@
 
 # *****************************************************************************
 # Backup SVN repositories
-docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z {{ repository_images.debian.image }}:{{ repository_images.debian.tag }} bash -c "rm -rf /tmp/import_export/SVN"
+docker run --rm -v {{ docker_backup_dir }}:/tmp/import_export:z debian:latest bash -c "rm -rf /tmp/import_export/SVN"
 mkdir -p -m u=rwx,g=rwx,o= {{ svn_docker_backup_dir }}
 docker run --rm \
     --volumes-from {{ svn_dv_name }} \
